qualification/tied_array_channelised_voltage/test_linearity.py

```diff
@@ -27,7 +27,7 @@
 from ..reporter import Reporter
 
 
-async def test_linearity(
+async def _test_linearity(
```

Comment:
Aha, I was thinking that when I did the initial review, just forgot to comment/ask. Thanks for changing it.